featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -749,10 +750,11 @@ def _necessary_columns(self, dataframe_name, feature_names):
         # relationships will come from this node.
         df = self.entityset[dataframe_name]
         index_columns = {col for col in df.columns
-                         if {'index', 'foreign_key', 'time_index'} & df[col].ww.semantic_tags}
+                         if {'index', 'foreign_key', 'time_index'} & df.ww[col].ww.semantic_tags}
```

Comment:
I _think_ that it'll generally be faster to do `df.ww.semantic_tags[col]` than going through the getitem (haven't fully confirmed this ever, but it means we don't have to go through woodwork column accessor init)

Reply:
Makes sense to me. I'll update.